cpp/cmake/thirdparty/get_treelite.cmake

```diff
@@ -35,11 +35,25 @@ function(find_and_configure_treelite)
     )
 
     set(Treelite_ADDED ${Treelite_ADDED} PARENT_SCOPE)
-    set(Treelite_SOURCE_DIR ${Treelite_SOURCE_DIR} PARENT_SCOPE)
 
-    if(Treelite_ADDED AND NOT TARGET treelite::treelite_static)
-        add_library(treelite::treelite_static ALIAS treelite_static)
-        add_library(treelite::treelite_runtime_static ALIAS treelite_runtime_static)
+    if(Treelite_ADDED)
```

Comment:
Should this be made not conditional on treelite as we were talking about offline?

Reply:
Turns out `Treelite_SOURCE_DIR` and `Treelite_BINARY_DIR` aren't populated if `Treelite_ADDED` is false.